### PR TITLE
Add an image showing stream processing with and without rw.

### PR DESCRIPTION
**Pull request by ShanlanLi**
## Info

- **Description**
- Two major changes:

  - [Add an image showing stream processing with and without rw to What is RisingWave?]
  - [Add a section called "Real-time pipeline orchestration" to "RisingWave vs. Apache Flink: Which one to choose?"]

- **Notes**

  - [ Please check if the position where I put  "Real-time pipeline orchestration" is suitable.  Now I put it after the "Introduction" of RisingWave&Flink and before the "Architecture" of them.]
  - [And I added a simple summary of this part(the italicized part) to aligin with other parts in this chapter. Please check it too.]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ RisingWave specializes in providing **incrementally updated, consistent material
 
 <img
   src={require('./images/stream_processing_with_and_without_rw.jpeg').default}
-  alt="RisingWave Architecture"
+  alt="Stream Processing With And Without RisingWave"
```
Comment (CharlieSYH) [inline docs/intro.md:28]:
```suggestion
  alt="Stream processing with and without RisingWave"
```

File: versioned_docs/version-1.5/intro.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ RisingWave specializes in providing **incrementally updated, consistent material
 
 <img
   src={require('./images/stream_processing_with_and_without_rw.jpeg').default}
-  alt="RisingWave Architecture"
+  alt="Stream Processing With And Without RisingWave"
```
Comment (CharlieSYH) [inline versioned_docs/version-1.5/intro.md:28]: same here

**Review by CharlieSYH — APPROVED** [review]

**Review by CharlieSYH — APPROVED** [review]
The rest LGTM

**Review by hengm3467 — APPROVED** [review]
LGTM. Thanks for the solid work! @ShanlanLi

**Pull request merged**